Source/UINavigation/Public/UINavInputDisplay.h

```diff
@@ -37,6 +38,10 @@ class UINAVIGATION_API UUINavInputDisplay : public UUserWidget
 	UFUNCTION(BlueprintCallable, Category = "InputDisplay")
 	void SetInputAction(UInputAction* NewAction, const EInputAxis NewAxis, const EAxisType NewScale);
 
+	// Locks to a specific input type. If set to None, it is automatically detected
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category="InputDisplay")
```

Comment:
I don't like that this is BlueprintReadWrite. I'd rather only have this be EditAnywhere and to most BlueprintReadOnly

Reply:
Yeah EditAnywhere is sufficient for our use case too. Removed Blueprint access entirely.